docs/requirements.txt

```diff
@@ -21,9 +21,8 @@ speechbrain==0.5.13
 huggingface_hub==0.11.1
 fasttext-wheel==0.9.2
 torch==1.13.1
-skorch==0.12.1
 torchvision==0.14.1
 torchaudio==0.13.1
 timm==0.6.12
 datasets>=2.9.0
-cleanvision
+cleanvision>=0.3.1
```

Comment:
Now we're tracking different version bounds in different places.

https://github.com/cleanlab/cleanlab/blob/14d1a86d4375beb17168eafdea88c84fd73c9a65/setup.py#L37

Can't we make `.[all]` an entry in this file? Maybe the entry can be editable? Something like:

```
-e ".[all]"
```

It would mean we'd have to update the docs/README.md.

Reply:
Sure, that would be easier to maintain.

Reply:
I think we can safely remove the "cleanvision" line if the editable install works, but not things like datasets.
The former is never used/imported in the tutorials, while the latter is used for some loading some datasets.